dev-docs/examples/intercept-banner-not-for-prod.js

```diff
@@ -0,0 +1,15 @@
+pbjs.que.push(() => {
+    pbjs.setConfig({
```

Comment:
```suggestion
window.pbjs = window.pbjs || { que: [] };
pbjs.que.push(() => {
    pbjs.setConfig({
```

I'll probably do the usually ceremony to avoid any conflicts